src/parser/hunter/survival/modules/spells/conduits/StrengthOfThePack.tsx

```diff
@@ -23,12 +23,12 @@ class StrengthOfThePack extends Analyzer {
 
   constructor(options: any) {
     super(options);
-    this.active = false;
+    this.active = this.selectedCombatant.hasConduitBySpellID(SPELLS.STRENGTH_OF_THE_PACK_CONDUIT.id);
     if (!this.active) {
       return;
     }
 
-    this.conduitRank = 1; //TODO: Find out the proper way of parsing conduit ranks
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.STRENGTH_OF_THE_PACK_CONDUIT.id);
```

Comment:
This feels a little bit repetitive because conduitRankBySpellID also checks if the conduit is active

```typescript
super(options);
let rank = this.selectedCombatant.conduitRankBySpellID(SPELLS.STRENGTH_OF_THE_PACK_CONDUIT.id);
if(!rank) {
  this.active = false;
  return;
}
this.conduitRank = rank;
```

Purely stylistic though, not something id request changes for if you disagree

Reply:
I agree on the spec specific ones, but on the covenant ones I can't do it without also allowing conduitRank to be a boolean so those stayed hasCovenant && hasConduit